examples/thermomechanics/thermal_deformation.cpp

```diff
@@ -114,12 +114,12 @@ void thermalDeformationExample( const std::string filename )
     //                      Outputs
     // ====================================================
     // Output displacement along the x-axis
-    createDisplacementProfile( MPI_COMM_WORLD, num_cells[0], 0,
-                               "xdisplacement_profile.txt", *particles );
+    createDisplacementProfile( MPI_COMM_WORLD, "xdisplacement_profile.txt",
```

Comment:
We need to modify these to have:
y-displacement along the x-axis
y-displacement along the y-axis

displacement magnitude along the x-axis
displacement magnitude along the y-axis